partner_financial_risk/models/res_company.py

```diff
@@ -2,12 +2,22 @@
 # © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import fields, models
+from openerp import api, fields, models
 
 
 class ResCompany(models.Model):
     _inherit = 'res.company'
 
     invoice_unpaid_margin = fields.Integer(
         string="Maturity Margin",
-        help="Days after due date to set an invoice as unpaid")
+        help="Days after due date to set an invoice as unpaid."
+             "The change of this field recompute all partners risk,"
+             "be patient.")
+
+    @api.multi
+    def write(self, vals):
+        res = super(ResCompany, self).write(vals)
```

Comment:
Isn't there an specific method for applying changes in this screen?

Reply:
It can be written from settings or from company, although it is not currently in the company view.